### PR TITLE
Rebuild fonts to fix #2

### LINKED ISSUE #2: IE8 displaying empty boxes

### PR DESCRIPTION
**Pull request by contolini**
See discussion at #2.

### CODE REVIEW

**Comment by ascott1** [comment]
Thanks for saving my sanity @contolini!

File: demo/index.html [hunk 1]
```diff
@@ -19,7 +19,18 @@
...
+    <script src="static/demo/html5shiv-printshiv.js"></script>
+    <script>
+      var cfHead = document.getElementsByTagName('head')[0],
+        cfStyle = document.createElement('style');
+      cfStyle.type = 'text/css';
+      cfStyle.styleSheet.cssText = ':before,:after{content:none !important}';
+      cfHead.appendChild(cfStyle);
+      setTimeout(function(){
+        cfHead.removeChild(cfStyle);
+      }, 100);
```
Comment (himedlooff) [inline demo/index.html:31]: Why does this say 100 and cf-component-demo says [300](https://github.com/cfpb/cf-component-demo/blob/master/raw/index.jade#L52)?

Comment (contolini) [inline demo/index.html:31]: I increased the delay to 300ms to play it safe. I must have built cf-icons before changing the delay here.

Comment (himedlooff) [inline demo/index.html:31]: Ok, thanks!

**Comment by himedlooff** [comment]
In src/fonts/cf-icons.svg line #7 says `<font id="cfpb_miniconsregular" horiz-adv-x="1792" >`. According to @ascott1 the ID should be **CFPBMinicons**. First, I'm not even sure what this even does, can somebody fill me in on why (if at all) this ID is important? And if it needs to be updated is this something that needs to be changed in the application that generates the fonts? Or is it something we can simply change in the SVG file? And what about the other file types, do they need to be checked (maybe this is only an SVG thing)? Such confuse.

BTW, I only noticed this because docs is using a different version of the SVG than src and demo. Once this is resolved we should make sure that docs and demo are rebuilt.

**Comment by ascott1** [comment]
Font files have three names (font name, font id, and something else). There have been posts noting that IE can choke when these three things aren't the same. I'm sure the SVG id is just generated when we generate the different file formats. Last week we tried to normalize this across all of the files as CFPBMinicons. It's possible that when @contolini ran this through the Font Squirrel generator, it changed the ID.

**Comment by himedlooff** [comment]
So it needs to be regenerated then?

**Comment by ascott1** [comment]
maybe? If this one is working, we should probably leave well enough alone though...

**Comment by himedlooff** [comment]
Ok, then at the very least we should re-run `grunt vendor` and the default Grunt task so that demo and docs are in sync. I will open a PR for this.

**Pull request merged**